pkg/crosscluster/streamclient/client_test.go

```diff
@@ -326,3 +326,110 @@ func TestStreamClientAppName(t *testing.T) {
 	expectAppName(t, "$ internal repstream")
 	expectAppName(t, "$ internal repstream job id=1337", WithStreamID(1337))
 }
+
+func TestInlineSSLCertParsing(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	// These are valid but totally unused certs and keys.
```

Comment:
nit: can you document the command you used to generate the certs in the comment?

Reply:
Well I technically just went to a random cert generator online 😅 I'll look up a proper command though